### PR TITLE
CRM-20764 Allow clean URLs with CiviCRM on Backdrop

### PR DESCRIPTION
**Pull request by herbdool**
Addresses https://issues.civicrm.org/jira/browse/CRM-20764

### CODE REVIEW

**Comment by herbdool** [comment]
@totten @colemanw here's hoping this is a fairly simple fix.

**Comment by seamuslee001** [comment]
@herbdool Herb can you rebase this branch against upstream master by doing git rebase -i upstream/master and just picking off commit e4bd329

File: templates/CRM/common/civicrm.settings.php.template [hunk 1]
```diff
@@ -443,6 +443,9 @@ if (!defined('CIVICRM_CLEANURL')) {
   if ( function_exists('variable_get') && variable_get('clean_url', '0') != '0') {
     define('CIVICRM_CLEANURL', 1 );
   }
+  elseif { function_exists('config_get') && config_get('system.core', 'clean_url') != 0) {
```
Comment (seamuslee001) [inline templates/CRM/common/civicrm.settings.php.template]: @herbdool there is also a php error here 
```
PHP Parse error:  syntax error, unexpected '{', expecting '(' in /home/jenkins/buildkit/build/core-10674-4ayju/sites/default/civicrm.settings.php on line 464
```

**Review by seamuslee001 — CHANGES_REQUESTED** [review]

**Comment by herbdool** [comment]
@seamuslee001 thanks, I've attempted your request and fixed the syntax error.

**Comment by monishdeb** [comment]
@herbdool there's a false merge commits in the PR (see the 2nd commit). It avoid that, always rebase your current branch then rebase. Here are the steps to do  that : 
1. cd <civicrm directory>
2. Then rebase
```
git fetch upstream master
git pull --rebase upstream master
``` 
3. ```git push -f herbdool crm20764```

**Comment by herbdool** [comment]
@monishdeb how about now?

**Comment by monishdeb** [comment]
Perfect 😃👍

File: templates/CRM/common/civicrm.settings.php.template (hunk 1 above)
Comment (laryn) [inline templates/CRM/common/civicrm.settings.php.template]: That first curly bracket should be a regular bracket:
`elseif ( function_exists('config_get') && config_get('system.core', 'clean_url') != 0) {`

**Comment by mlutfy** [comment]
@monishdeb @herbdool Any reason why this hasn't been merged yet?

**Comment by herbdool** [comment]
@mlutfy I'm not sure why it hasn't been merged. I think it's ready.

**Comment by seamuslee001** [comment]
Code looks good to me

**Comment by mlutfy** [comment]
OK, taking the liberty of merging this one, based on the comments/reviews.

**Pull request merged**